txclib/utils.py

```diff
@@ -105,27 +111,57 @@ def determine_charset(response):
 
 def make_request(method, host, url, username, password, fields=None,
                  skip_decode=False):
-    charset = None
-    if host.lower().startswith('https://'):
-        connection = urllib3.connection_from_url(
-            host,
-            cert_reqs=CERT_REQUIRED,
+    # Initialize http and https pool managers
+    num_pools = 1
+    managers = {}
+    if "http_proxy" in os.environ:
+        proxy_url = os.environ["http_proxy"]
+        managers["http"] = urllib3.ProxyManager(
+            proxy_url=proxy_url,
+            proxy_headers={"User-Agent": user_agent_identifier()},
+            num_pools=num_pools
+        )
+    else:
+        managers["http"] = urllib3.PoolManager(num_pools=num_pools)
+
+    if "https_proxy" in os.environ:
+        proxy_url = os.environ["https_proxy"]
+        managers["https"] = urllib3.ProxyManager(
+            proxy_url=proxy_url,
+            proxy_headers={"User-Agent": user_agent_identifier()},
+            num_pools=num_pools,
+            cert_reqs=ssl.CERT_REQUIRED,
```

Comment:
@mkcello96 Why are you using  `ssl.CERT_REQUIRED` here above we've imported and used `txclib.config.CERT_REQUIRED`